go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/btcsuite/btcd v0.22.1
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
-	github.com/cosmos/cosmos-sdk v0.45.4
+	github.com/cosmos/cosmos-sdk v0.45.5-0.20220523154235-2921a1c3c918
```

Comment:
is this a stable release?

Reply:
it's the `v0.45.x` release branch, but not tagged as a new release yet.

Reply:
why not use the latest?

Reply:
do you mean upgrade to 0.46? I think 0.46 will have some breaking API changes, maybe better to do that separately.

Reply:
latest as in `v0.45.4`, can we update it to use that one, or is there a change that was required for this to work?

Reply:
https://github.com/cosmos/cosmos-sdk/pull/12017
yeah, there's fix there.